sharding-transaction-spi-impl/sharding-transaction-base-spi-impl/sharding-transaction-base-saga/src/main/java/io/shardingsphere/transaction/saga/utils/JDBCUtil.java

```diff
@@ -47,9 +47,7 @@
      */
     public static List<Map<String, Object>> executeQuery(final Connection connection, final String sql, final Collection<Object> params) throws SQLException {
         List<Map<String, Object>> result = new ArrayList<>();
-        PreparedStatement preparedStatement = null;
-        try {
-            preparedStatement = connection.prepareStatement(sql);
+        try (PreparedStatement preparedStatement = connection.prepareStatement(sql);) {
             fillParamter(preparedStatement, params);
```

Comment:
method `fillParamter` should name as `fillParamters`

Reply:
i remove it